commands.go

```diff
@@ -146,6 +147,41 @@ func quipMessages(s *discordgo.Session, m *discordgo.MessageCreate) {
 	}
 }
 
+func featureRequestResponse(s *discordgo.Session, m *discordgo.MessageCreate) {
+	ctx := context.Background()
+	var span *trace.Span
+	me := hnydiscordgo.MessageEvent{Message: m.Message, Context: ctx}
+
+	ctx, span = hnydiscordgo.StartSpanOrTraceFromMessage(&me, s)

```

Comment:
This isn't needed as it's handled in the initial routing code, it's only really needed in the Adilio functions as they deviate from the normal bot command logic by not needing a ! to trigger them. Just creating a new span for the command and adding any relevant fields is needed. I'd think we'd want to add the random number and the possible choice as fields, the response could be added but it's handled by the sendResponse function so not really necessary.

Reply:
👍 passed in the existing context (as per getTime)